modules/templates/helper.go

```diff
@@ -343,7 +343,7 @@ func ReplaceLeft(s, oldS, newS string) string {
 
 	// allocating space for the new string
 	curLen := n*newLen + len(s[i:])
-	replacement := make([]byte, curLen, curLen)
+	replacement := make([]byte, curLen)
```

Comment:
Why change this line?

Reply:
Because the `cap` of a new slice is optional and if omitted, it is the same as the provided `len`. So since it is the same here, there's no need to specify both.